modules/customize-certificates-add-service-serving-apiservice.adoc

```diff
@@ -0,0 +1,71 @@
+// Module included in the following assemblies:
+//
+// * authentication/certificates/service-serving-certificate.adoc
+
+[id="add-service-certificate-apiservice_{context}"]
+= Add the service CA bundle to an APIService
+
+////
+
+// TODO: intro + important note?
+
+A Pod can access the service CA certificate by mounting a ConfigMap that
+is annotated with `service.beta.openshift.io/inject-cabundle=true`.
+Once annotated, the cluster automatically injects the service CA
+certificate into the `service-ca.crt` key on the ConfigMap. Access to
+this CA certificate allows TLS clients to verify connections to
+services using service serving certificates.
+
+[IMPORTANT]
+====
+After adding this annotation to a ConfigMap all existing data in it is
+deleted. It is recommended to use a separate ConfigMap to contain the
+`service-ca.crt`, instead of using the same ConfigMap that stores your
+Pod's configuration.
+====
+////
+
+.Procedure
+
+. Annotate the APIService with `service.beta.openshift.io/inject-cabundle=true`.
++
+----
+$ oc annotate apiservice <apiservice-name> \//<1>
+     service.beta.openshift.io/inject-cabundle=true
+----
+<1> Replace `<apiservice-name>` with the name of the APIService to annotate.
++
+////
```

Comment:
Only configmaps use the key 'service-ca.crt'. Other resources - apiservice, crd, webhook configs - have the ca bundle injected into a type-specific field.

Reply:
@marun So we don't need this note at all then for these that use the caBundle field, correct?